README.md

```diff
@@ -69,59 +72,73 @@
 
-With a working `anaconda` installation, install the `bioconda` and `conda-forge` channels:
+`pip` will install `pyani` and its Python dependencies, but not the third-party tools.
 
 ```bash
-conda config --add channels defaults
-conda config --add channels bioconda
-conda config --add channels conda-forge
+pip3 install pyani
 ```
 
-Then install `pyani`:
+### Third-party tools
+
+Three alignment packages are required, to use all of `pyani`'s methods: `mummer`, `BLAST+`, and legacy `BLAST`.
+
+The simplest route to obtaining these tools is to use `conda`/`bioconda`:
 
 ```bash
-conda install pyani
+conda install mummer blast legacy-blast -y
 ```
 
-**NOTE:** The current `conda` version is *behind* the development version and has a different command-line interface. Documentation for the `conda` version (0.2.x) can be found at the link below:
-
-- [https://github.com/widdowquinn/pyani/blob/master/README_v_0_2_x.md](https://github.com/widdowquinn/pyani/blob/master/README_v_0_2_x.md)
+But they can also be installed by following instructions from the tools' own websites.
 
+- **BLAST+** (for `anib`) [ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/)
+- **legacy BLAST** (for `aniblastall`) [ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/)
+- **MUMmer** (for `anim`) [http://mummer.sourceforge.net/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/)
 
-### Installing legacy BLAST
+### NOTE: Installing legacy BLAST
 
 The NCBI legacy `BLAST` package is deprecated and not recommended. However, if you wish to use `pyani blastall` or the `ANIblastall` method with the legacy `pyani` interface, you will require a locally-installed copy of this package. This is one of the packages linked in the `requirements-thirdparty.txt` file.
 
 ## Citing `pyani`
 
 A guide to citing `pyani` is included in the file [`CITATIONS`](CITATIONS)
 
-## Documentation
+## Documentation (v0.3)
 
 This `README.md` file provides a quick overview and walkthrough for **THE DEVELOPMENT VERSION (v0.3+) OF `pyani`**, and full documentation can be found at the link below:
 
 - [`pyani` v0.3+ Documentation](https://readthedocs.org/projects/pyani/)
 
-### Older documentation
+**THIS README AND THE DOCUMENTATION AT `READTHEDOCS` REFERS TO A DEVELOPMENT VERSION OF `PYANI` (v0.3+). IT HAS A DIFFERENT COMMAND-LINE INTERFACE THAN THE STABLE `PYANI` VERSION (v0.2.x).**
+
+**THE STABLE VERSION OF `PYANI` (v0.2) DOES NOT HAVE THE `pyani` COMMAND**
+
+### Older documentation (v0.2)
 
-If you are using an older version of `pyani` (v0.2.x), then please note that the command-line API has changed, and documentation for this version can be found at the following page:
+If you are using the stable version of `pyani` (v0.2.x), then please note that the command-line API has changed, and documentation for this version can be found at the following page:
```

Comment:
This references a change in the command-line API for v0.2.x? The wording is a bit ambiguous. It could be taken to mean an intra-version change, or to refer to the difference between version 2 and version 3.

Reply:
I think `the stable` is clearer only until v0.3 is the stable version, but `an older` is unnecessary and could be removed.

Reply:
I'm not sure that addresses my question, which is caused by the second phrase. I only know the CLI for v0.2 and v0.3 as they are now, so I am unclear whether the referenced change is:
1. the one between v0.2 and v0.3, and you're warning them that the documentation they see is not applicable to users of the stable version; or
1. if you mean a change from a previous version of v0.2 and the current, stable version, and you're warning people they may need to update their scripts if they update `pyani` to the current v0.2.

I think, given the wording, the second option seems more likely, but I don't know if there were any changes to the CLI in v0.2.